webrtc-stats.html

```diff
@@ -1859,23 +1841,56 @@ <h2>
               </dt>
               <dd>
                 <p>
-                  Only valid for video. It represents the nominal FPS value before the degradation
+                  Represents the nominal FPS value before the degradation
                   preference is applied. It is the number of complete frames in the last second.
                   For sending tracks it is the current captured FPS and for the receiving tracks it
                   is the current decoding framerate.
                 </p>
               </dd>
+            </dl>
+          </section>
+        </div>
+      </section>
+      <section id="vsstats-dict*">
+        <h3>
+          <dfn>RTCVideoSenderStats</dfn> dictionary
+        </h3>
+        <p>
+          An RTCVideoSenderStats object represents the stats about one video sender of a
+          PeerConnection object for which one calls getStats.
+        </p>
+        <p>
+          It appears in the stats as soon as the sender is added by either addTrack or
+          addTransceiver, or by media negotiation.
+        </p>
+        <div>
+          <pre class="idl">dictionary RTCVideoSenderStats : RTCVideoStreamTrackStats {
+             unsigned long       framesCaptured;
+             unsigned long       framesSent;
+             unsigned long       keyFramesSent;
+};</pre>
+          <section>
+            <h2>
+              Dictionary <a class="idlType">RTCVideoSenderStats</a> Members
+            </h2>
+            <dl data-link-for="RTCVideoSenderStats" data-dfn-for="RTCVideoSenderStats"
+            class="dictionary-members">
               <dt>
                 <dfn><code>framesCaptured</code></dfn> of type <span class=
                 "idlMemberType"><a>unsigned long</a></span>
               </dt>
               <dd>
                 <p>
-                  Only valid for local video. It represents the total number of frames captured for
-                  this MediaStreamTrack, before encoding. For example, if this track represents a
-                  camera this is the number of frames produced by the camera for this track, whose
-                  framerate could vary due to hardware limitations or environmental factors such as
-                  lighting conditions.
+                  Represents the total number of frames captured, before encoding,
+                  for this RTCRtpSender (or for this MediaStreamTrack, if
```

Comment:
nit: Here and elsewhere: "...for this MediaStreamTrack _attachment_..."

Reply:
Right, though that was probably an existing issue? This PR mostly moves words around.